corehq/apps/export/dbaccessors.py

```diff
@@ -181,6 +181,19 @@ def _properly_wrap_export_instance(doc):
     return class_.wrap(doc)
 
 
+def get_default_export_settings_for_domain(domain):
+    from .models import DefaultExportSettings
+    from corehq.apps.accounting.models import Subscription, SoftwarePlanEdition
```

Comment:
I think this is the appropriate place for this db helper, but not sure if it is bad practice to import models from another app.

Reply:
I would call it `get_or_create_default_export_settings_for_domain`. Also I would lean toward this being a utility vs a dbaccessor as the dbaccessors are really meant for interfacing with couch <> SQL backends